astor/file_util.py

```diff
@@ -95,16 +95,16 @@ def __init__(self, cache=None):
 
     def __call__(self, codeobj):
         cache = self.cache
-        key = self.get_file_info(codeobj)
+        fname, lineno = self.get_file_info(codeobj)
+        key = (fname, codeobj.__name__)
         result = cache.get(key)
         if result is not None:
             return result
-        fname = key[0]
         cache[(fname, 0)] = mod_ast = self.parse_file(fname)
```

Comment:
So now key for, say `a.py`'s AST tree, is `('a.py', 0)`, but key for `foo()` in `a.py` is `('a.py', 'foo')`. I think it would be nice to have the same types of both elements in the tuple. I'll try to fix it.